### PR TITLE
ComputeRP 2022-03-01 REST API release

### PR DESCRIPTION
**Pull request by grizzlytheodore**
PR for ComputeRP 2022-03-01 REST API release
- This release, we broke up the compute.json swagger file into different resources
- grouped example files by each swagger files 
- cleaned up example file names
- before adding any new features, to check that no mistakes were made from breaking up the swagger files, we generated SDK to make sure there were no changes made to the generated SDK 

PRs from CRP for this release: 
https://github.com/Azure/azure-rest-api-specs/pull/18295
https://github.com/Azure/azure-rest-api-specs/pull/18295
https://github.com/Azure/azure-rest-api-specs/pull/18165
https://github.com/Azure/azure-rest-api-specs/pull/18592 <- breaking change approved, ARM reviewed
https://github.com/Azure/azure-rest-api-specs/pull/18433
https://github.com/Azure/azure-rest-api-specs/pull/18487 <- breaking change approved, ARM reviewed
https://github.com/Azure/azure-rest-api-specs/pull/18609
https://github.com/Azure/azure-rest-api-specs/pull/18581
https://github.com/Azure/azure-rest-api-specs/pull/18159 

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [x] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by mentat9** [comment]
@grizzlytheodore - Please study the materials and attest to that fact by checking this item:

**[ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). *This is required before you can request review from ARM API Review board*.**

**Comment by grizzlytheodore** [comment]
for ARM reviewer: I know we are suppose to have the first commit as setting up the branch, but we broke up the swagger files this release, and it was difficult to get it right on the first try. Apologies for having many commit for setting up branch. In the PR description up top, I have listed out all the features that are being added to this release and it's PRs. Please see.

**Comment by grizzlytheodore** [comment]
for ARM reviewer: I know we are suppose to have the first commit as setting up the branch, but we broke up the swagger files this release, and it was difficult to get it right on the first try. Apologies for having many commit for setting up branch. In the PR description up top, I have listed out all the features that are being added to this release and it's PRs. Please see.

**Comment by grizzlytheodore** [comment]
the breaking change flag shown for SDK JS is approved/explained/merged here https://github.com/Azure/azure-rest-api-specs/pull/18487

File: specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json [hunk 1]
```diff
@@ -0,0 +1,1059 @@
...
+  },
+  "definitions": {
+    "DedicatedHostGroupInstanceView": {
+      "properties": {
+        "hosts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DedicatedHostInstanceViewWithName"
+          },
+          "x-ms-identifiers": [],
```
Comment (mentat9) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json:766]: >"x-ms-identifiers": [] [](http://example.com/codeflow?start=10&length=22)

Looks like "name" was removed (as compared with 2021-11-01\compute.json. Was this intentional?

**Comment by mentat9** [comment]
@grizzlytheodore - I've reviewed everything and it looks good from ARM perspective, aside from one comment above.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json (hunk 1 above)
Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json]: this must have been missed. just synced it with Compute. Thank you @mentat9

**Comment by grizzlytheodore** [comment]
> @grizzlytheodore - Please study the materials and attest to that fact by checking this item:
> 
> **[ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). _This is required before you can request review from ARM API Review board_.**

checked! @mentat9  please approve ARM!

**Comment by grizzlytheodore** [comment]
@ArcturusZhang  please review. It is ready for merge!

**Comment by mentat9** [comment]
> > @grizzlytheodore - Please study the materials and attest to that fact by checking this item:
> > **[ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). _This is required before you can request review from ARM API Review board_.**
> 
> checked! @mentat9 . I've also addressed your comment. please approve ARM!

Done.

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**